client/qiskit_serverless/core/client.py

```diff
@@ -27,131 +27,20 @@
     ServerlessClient
 """
 # pylint: disable=duplicate-code
-import logging
-import warnings
-import os.path
-import os
-from dataclasses import dataclass
+from abc import ABC, abstractmethod
 from typing import Optional, List, Dict, Any, Union
 
-import ray
-import requests
-from ray.dashboard.modules.job.sdk import JobSubmissionClient
-from opentelemetry import trace
 from qiskit_ibm_runtime import QiskitRuntimeService
 
-from qiskit_serverless.core.constants import (
-    REQUESTS_TIMEOUT,
-    ENV_GATEWAY_PROVIDER_HOST,
-    ENV_GATEWAY_PROVIDER_VERSION,
-    ENV_GATEWAY_PROVIDER_TOKEN,
-    GATEWAY_PROVIDER_VERSION_DEFAULT,
-    IBM_SERVERLESS_HOST_URL,
-)
-from qiskit_serverless.core.files import GatewayFilesClient
-from qiskit_serverless.core.job import (
-    Job,
-    RayJobClient,
-    GatewayJobClient,
-    LocalJobClient,
-    BaseJobClient,
-    Configuration,
-)
+from qiskit_serverless.core.job import Job
 from qiskit_serverless.core.function import QiskitFunction
-from qiskit_serverless.core.tracing import _trace_env_vars
-from qiskit_serverless.exception import QiskitServerlessException
 from qiskit_serverless.utils import JsonSerializable
-from qiskit_serverless.utils.json import safe_json_request
 from qiskit_serverless.visualizaiton import Widget
 
 TIMEOUT = 30
 
 
-@dataclass
-class ComputeResource:
```

Comment:
Here we are removing the entire `ComputeResource` reference. This change was not annotated in the PR, can you add this too, @korgan00 ?

Also, from my part it's fine because I think we are not using it but @IceKhan13 WDYT?